src/core/server/ui_settings/utils.ts

```diff
@@ -7,10 +7,19 @@ import { UiSettingScope } from './types';
 
 export const CURRENT_USER_PLACEHOLDER = '<current_user>';
 export const CURRENT_WORKSPACE_PLACEHOLDER = '<current_workspace>';
+
+export const DASHBOARD_ADMIN_SETTINGS_ID = '_dashboard_admin';
+
+export const PERMISSION_CONTROLLED_UI_SETTINGS_WRAPPER_ID = 'permission-control-ui-settings';
+export const PERMISSION_CONTROLLED_UI_SETTINGS_WRAPPER_PRIORITY = 100;
```

Comment:
Nit: Better to leave some comment for the priority.

Reply:
Done